rsp/runtimes/bundles/org.jboss.tools.rsp.server.tomcat/src/main/resources/servers.json

```diff
@@ -268,7 +276,7 @@
 							"workingDirectory": "bin",
 							"mainType": "org.apache.catalina.startup.Bootstrap",
 							"programArgs": "start",
-							"vmArgs": "-Djava.util.logging.config.file=${server.home.dir}/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base=${server.home.dir} -Dcatalina.home=${server.home.dir} -Djava.io.tmpdir=${server.home.dir}/temp",
+							"vmArgs": "-Djava.util.logging.config.file=${server.home.dir}/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dcatalina.base=${server.base.dir} -Dcatalina.home=${server.home.dir} -Djava.io.tmpdir=${server.home.dir}/temp",
```

Comment:
I thought the config below should be changed from ${server.home.dir} into ${server.base.dir}, according to the Tomcat Official Document: _Advanced Configuration - Multiple Tomcat Instances Section_ in [Running.txt](https://tomcat.apache.org/tomcat-9.0-doc/RUNNING.txt)

![image](https://user-images.githubusercontent.com/33516065/164218851-67d944b9-8f78-4a0a-b585-d96262fabeb5.png)

-Djava.util.logging.config.file=${server.base.dir}/conf/logging.properties
-Djava.io.tmpdir=${server.home.dir}/temp